blockwork/build/transform.py

```diff
@@ -173,10 +230,11 @@ def run(self, ctx: "Context"):
             if exit_code:=container.invoke(ctx, invocation) != 0:
                 raise RuntimeError(f"Invocation `{invocation}` failed with exit code `{exit_code}`.")
 
+    
     def execute(self,
              ctx  : "Context",
              tools: ReadonlyNamespace["Version"],
-             iface: ReadonlyNamespace[Any]) -> Iterable["Invocation"]:
+             iface: ReadonlyNamespace[Any], /) -> Iterable["Invocation"]:
```

Comment:
What does the `/` do?

Reply:
Marks the preceding arguments as positional-only - in this case it is so the bind_execute call can bind a lambda function with different (shorter) argument names